src/client/gameui.cpp

```diff
@@ -76,6 +76,11 @@ void GameUI::init()
 	m_guitext_chat = gui::StaticText::add(guienv, L"", core::rect<s32>(0, 0, 0, 0),
 		//false, false); // Disable word wrap as of now
 		false, true, guiroot);
+	u16 chat_font_size = g_settings->getU16("chat_font_size");
```

Comment:
wouldn't it be better if the default value was empty instead of 0?

Reply:
This is an integer setting, hence setting this value to an empty string will not be possible using the `Settings` formspec.